Home-open event updates and links across events and multitab pages (#583)
HL7/fhircast-docs: input/pagecontent/4-4-multitab-considerations.md

### Launching A Context-Less Tab

Many applications can have a "home" or "default" tab that contains no clinical context, but may hold useful application features. In some cases other applications may want to subscribe to and be notified when another application has switched to the no context tab.

> Since we are inherently representing the lack of context, the event will not fully conform to the defined event naming syntax and will instead use a static name (similar to `UserLogout.html`).

Reviewer: The [Home-open event](3-2-5-Home-open.html) represents a lack of context.

> Since we are inherently representing the lack of context, the event will not fully conform to the defined event naming syntax and will instead use a static name (similar to `UserLogout.html`).

Reviewer: ```suggestion
The [Home-open event](3-2-5-Home-open.html) represents a lack of context.
```

### Launching A Context-Less Tab

Many applications feature a "home" or "default" tab that contains no clinical context but may offer useful application features. Sometimes, other applications may want to subscribe to and be notified when an application switches to this no-context tab. This is indicated by the [Home-open event](3-2-5-Home-open.html).

> Note that sending a [Home-open event](3-2-5-Home-open.html) merely signals that an application has switched to the no-context tab. It does not change the current context. For instance, if a specific patient is currently in context, sending a Home-open event does not imply that this patient is no longer in context.

Reviewer: ```suggestion
```
